### PR TITLE
French translation for the bootcamp description.

### PR DESCRIPTION
**Pull request by ccossou**
Here is, in 4 new files, the translations for the english description. A "_fr" has been added to the original name of the file.

### CODE REVIEW

File: bootcamps/_content_fr.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<p>
+  <strong>Contenu:</strong> Le programme de la session d'entraînement contiendra :
```
Comment (agramfort) [inline bootcamps/_content_fr.html]: Le programme de la formation ...

File: bootcamps/_what_fr.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<p>
+  <strong>Quoi:</strong>
+  Notre but est d'aider les scientifiques et ingénieurs à être plus 
+  productifs en leur enseignant les compétences informatiques de 
+  base comme le design de program, la gestion de versions, l'art de
```
Comment (agramfort) [inline bootcamps/_what_fr.html]: programme, le suivi de versions,

File: bootcamps/_what_fr.html [hunk 3]
```diff
@@ -0,0 +1,15 @@
+<p>
+  <strong>Quoi:</strong>
+  Notre but est d'aider les scientifiques et ingénieurs à être plus 
+  productifs en leur enseignant les compétences informatiques de 
+  base comme le design de program, la gestion de versions, l'art de 
+  tester un code et l'automatisation des tâches. Lors de cette 
+  session d'entrainement de deux jours, des tutoriels courts seront
```
Comment (agramfort) [inline bootcamps/_what_fr.html]: s/session d'entrainement/formation

File: bootcamps/_what_fr.html [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+  base comme le design de program, la gestion de versions, l'art de 
+  tester un code et l'automatisation des tâches. Lors de cette 
+  session d'entrainement de deux jours, des tutoriels courts seront 
+  entrecoupés d'exercices où chacun pourra mettre en pratique 
+  immédiatement. Les participants seront encouragés à s'entraider et 
+  à appliquer ce qu'ils ont appris à leurs propres problèmes de 
+  recherche pendant et en dehors des sessions. Il y aura des 
+  sessions de suivi en ligne pendant 6 à 8 semaines afin de 
+  compléter et renforcer les bases enseignées lors de la session 
+  d'entraînement.
```
Comment (agramfort) [inline bootcamps/_what_fr.html:14]: s/session d'entrainement/formation

File: bootcamps/_who_fr.html [hunk 5]
```diff
@@ -0,0 +1,8 @@
+<p>
+  <strong>Qui:</strong>
+  La session est destinée aux étudiants de 3e cycle et autres
```
Comment (agramfort) [inline bootcamps/_who_fr.html]: La formation

File: bootcamps/_who_fr.html [hunk 6]
```diff
@@ -0,0 +1,8 @@
+<p>
+  <strong>Qui:</strong>
+  La session est destinée aux étudiants de 3e cycle et autres 
+  scientifiques qui ont les bases de la programmation (comme les 
+  boucles, conditions, tableaux et fonctions) mais qui ont besoin 
+  d'aide pour mettre ce savoir en pratique pour les aider dans leur
```
Comment (agramfort) [inline bootcamps/_who_fr.html]: d'aide pour mettre ce savoir en pratique de la manière la plus productive possible.

**Comment by agramfort** [comment]
besides looks go to me. Let me know when you pushed your updates.

**Comment by agramfort** [comment]
LGTM I'll let @gvwilson merged if web formatting is correct.

**Comment by pipitone** [comment]
Fixed in #252

**Pull request closed without merging**